src/build.js

```diff
@@ -1,148 +1,200 @@
-		'wgxpath.install.js',
-		'common_ie.js',
-		REPOSITORY_URL + '/masterlist.js',
-		'iframe_ie.js'
-	],
-	fn = jade.compileFile('iframe.jade');
-
-fs.writeFile('../iframe.html', fn(iframeScripts), function(err) {
-    if(err) {
-        return console.log(err);
+var minifyAll = function() {
+
+    console.log("Generating js files to output " + parameters.output + " and compression " + parameters.compressorType);
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: injectScripts.concat(['bookmarklet/inject_base.js']),
+        fileOut: parameters.output + 'inject.js',
+        callback: function (err, min) {
+            handleMessage("inject.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: injectScripts.concat(injectIEScripts, ['bookmarklet/inject_base.js']),
+        fileOut: parameters.output + 'inject_ie.js',
+        callback: function (err, min) {
+            handleMessage("inject_ie.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: commonScripts,
+        fileOut: parameters.output + 'common.js',
+        callback: function (err, min) {
+            handleMessage("common.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: commonScripts.concat(commonIEScripts),
+        fileOut: parameters.output + 'common_ie.js',
+        callback: function (err, min) {
+            handleMessage("common_ie.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: iframeScripts.concat(['bookmarklet/iframe_base.js']),
+        fileOut: parameters.output + 'iframe.js',
+        callback: function (err, min) {
+            handleMessage("iframe.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: iframeScripts.concat(iFrameIEScripts, ['bookmarklet/iframe_base.js']),
+        fileOut: parameters.output + 'iframe_ie.js',
+        callback: function (err, min) {
+            handleMessage("iframe_ie.js", err);
+        }
+    });
+
+    new compressor.minify({
+        type: parameters.compressorType,
+        language: 'ECMASCRIPT5',
+        fileIn: 'PME.js',
+        fileOut: parameters.output + 'PME.js',
+        callback: function (err, min) {
+            handleMessage("PME.js", err);
+        }
+    });
+
+};
+
+var parseArguments = function() {
+    for(var i = 0; i < process.argv.length; i++) {
+        var arg = process.argv[i];
+        var val = process.argv[++i];
+        console.log(arg, val);
+        switch(arg) {
+            case '-o': //Output folder, default is "../"
```

Comment:
Isn't the default ../output/ ?   See line 6

Reply:
Yes, you are correct.